### PR TITLE
Enable new Fluentbit metric exposure

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Enable new fluentbit metric exposure with prometheus output plugin
- Add new input plugin `fluentbit_metrics` and configuration
- Add new output plugin `prometheus_exports` and configuration

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/17164

### CODE REVIEW

File: internal/fluentbit/config/builder/config_builder.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func BuildFluentBitConfig(pipeline *telemetryv1alpha1.LogPipeline, defaults Pipe
 	sb.WriteString(createKubernetesMetadataFilter(pipeline))
 	sb.WriteString(createLuaDedotFilter(pipeline))
 	sb.WriteString(createOutputSection(pipeline, defaults))
-
+	sb.WriteString(createPrometheusMetricOutput("internal_metrics"))
```
Comment (chrkl) [inline internal/fluentbit/config/builder/config_builder.go:38]: This configuration is pipeline specific and would repeat the config snippet per LogPipelien. I think `fluentbit.MakeConfigMap` would be the better place for this.

Comment (hisarbalik) [reply internal/fluentbit/config/builder/config_builder.go]: moved

**Review by skhalash — CHANGES_REQUESTED** [review]
Please add an e2e test. We have already tests for trace and metric gateway that call the metrics endpoint

File: test/e2e/testkit/otlp/matchers/prometheus_matcher.go
Comment (skhalash) [inline test/e2e/testkit/otlp/matchers/prometheus_matcher.go:1]: Small comment - this matcher has nothing to do with OTLP. Maybe let's move the `matchers` directory from `otlp`?

**Review by skhalash — DISMISSED** [review]

File: test/e2e/testkit/otlp/matchers/prometheus_matcher.go
Comment (hisarbalik) [reply test/e2e/testkit/otlp/matchers/prometheus_matcher.go:1]: matchers package moved to the testkit root package

**Review by skhalash — APPROVED** [review]

**Pull request merged**